test/conformance/kuttl/validate/patterns/compare-arrays/README.md

```diff
@@ -0,0 +1,11 @@
+# ## Description
+
+This test creates a policy to validate resources with patterns comparing arrays.
```

Comment:
This is specifically about comparing more than one item in the array so mention that. It's important.

Reply:
Done.